### PR TITLE
Fix text parsing

### PR DESCRIPTION
**Pull request by dnfield**
We're currently ignoring the attributes on `<text>` when there's nested `tspan`s. 

Fixes https://github.com/dnfield/flutter_svg/issues/870 with a reduced test case.
Fixes https://github.com/dnfield/flutter_svg/issues/872
Fixes https://github.com/dnfield/flutter_svg/issues/763
Fixes https://github.com/dnfield/flutter_svg/issues/762 (which is probably a duplicate of 763).

### CODE REVIEW

**Review by jonahwilliams — APPROVED** [review]
LGTM

File: packages/vector_graphics_compiler/lib/src/svg/masking_optimizer.dart [hunk 1]
```diff
@@ -21,22 +21,18 @@ class _Result {
 /// Converts a vector_graphics PathFillType to a path_ops FillType.
 path_ops.FillType toPathOpsFillTyle(PathFillType fill) {
   switch (fill) {
-    case PathFillType.evenOdd:
-      return path_ops.FillType.evenOdd;
+    case PathFillType.evenOdd: return path_ops.FillType.evenOdd;
```
Comment (jonahwilliams) [inline packages/vector_graphics_compiler/lib/src/svg/masking_optimizer.dart:24]: FYI I believe this dartfmt change was reverted

**Review by jonahwilliams — APPROVED** [review]
LGTM

File: packages/vector_graphics_compiler/lib/src/svg/masking_optimizer.dart (hunk 1 above)
Comment (dnfield) [reply packages/vector_graphics_compiler/lib/src/svg/masking_optimizer.dart:24]: Yeah, hasn't rolled into flutter master as of this commit.

**Comment by dnfield** [comment]
I'm going to add a few more tests to this. I'm also seeing if I can get whitespace handling to be more like what the browsers do.

**Pull request merged**